Sources/ProjectDescription/Dependencies/CarthageDependencies.swift

```diff
@@ -4,11 +4,15 @@ import Foundation
 public struct CarthageDependencies: Codable, Equatable {
     /// List of dependencies that will be installed using Carthage.
     public let dependencies: [Dependency]
+    /// Set up Carthage's options.
+    public let options: Options
 
     /// Creates `CarthageDependencies` instance.
     /// - Parameter dependencies: List of dependencies that can be installed using Carthage.
-    public init(_ dependencies: [Dependency]) {
+    /// - Parameter options: Set up Carthage's options.
+    public init(_ dependencies: [Dependency], _ options: Options = .init()) {
```

Comment:
let's keep the options mail
```suggestion
    public init(_ dependencies: [Dependency], options: Options = .init()) {
```